ppr-ui/tests/unit/Breadcrumb.spec.ts

```diff
@@ -268,4 +269,78 @@ describe('Breadcrumb component tests', () => {
     tombstoneBreadcrumbAmendment[1].text = breadcrumbsTitles[userRoleProductCode]
     expect(breadcrumbs.at(1).text()).toContain('Staff')
   })
+
+  it('renders on Mhr Registration: staff', () => {
+    wrapper = createComponent(RouteNames.MHR_REGISTRATION)
+
+    const userRoleProductCode = getRoleProductCode(store.getUserRoles, [ProductCode.MHR])
+    expect(wrapper.find(backBtn).exists()).toBe(true)
+
+    const breadcrumbs = wrapper.findAll(getTestId('breadcrumb-item'))
+    expect(breadcrumbs.length).toBe(tombstoneBreadcrumbRegistration.length)
+    expect(breadcrumbs.at(1).text()).toContain('Staff')
+
+    tombstoneBreadcrumbRegistration[1].text = breadcrumbsTitles[userRoleProductCode]
+    for (let i = 0; i < tombstoneBreadcrumbRegistration.length; i++) {
+      expect(breadcrumbs.at(i).text()).toContain(wrapper.vm.handleStaff(tombstoneBreadcrumbRegistration[i].text))
+    }
+  })
+
+  it('renders on Mhr Registration: Manufactuer - Only Mhr', async () => {
+    // Set up
+    await store.setAuthRoles(mockedManufactuerAuthRoles)
+
+    defaultFlagSet['mhr-ui-enabled'] = true
+
+    store.setUserProductSubscriptionsCodes([ProductCode.MHR])
+
+    await nextTick()
+
+    wrapper = createComponent(RouteNames.MHR_REGISTRATION)
+
+    const userRoleProductCode = getRoleProductCode(store.getUserRoles, [ProductCode.MHR])
+    expect(wrapper.find(backBtn).exists()).toBe(true)
+
+    const breadcrumbs = wrapper.findAll(getTestId('breadcrumb-item'))
+    expect(breadcrumbs.length).toBe(tombstoneBreadcrumbRegistration.length)
+    expect(breadcrumbs.at(1).text()).toContain('My Manufactured Home Registry')
+
+    tombstoneBreadcrumbRegistration[1].text = breadcrumbsTitles[userRoleProductCode]
+    for (let i = 0; i < tombstoneBreadcrumbRegistration.length; i++) {
+      expect(breadcrumbs.at(i).text()).toContain(wrapper.vm.handleStaff(tombstoneBreadcrumbRegistration[i].text))
+    }
+
+    // Tear down
+    defaultFlagSet['mhr-ui-enabled'] = false
+    store.setUserProductSubscriptionsCodes([])
+  })
+
+  it('renders on Mhr Registration: Manufactuer - MHR and PPR', async () => {
```

Comment:
Typo here. Not a big deal, but if someone will be doing a global search for "Manufacturer" it won't come up.

Reply:
Thanks!